### ISSUE TITLE
Chat not working

### ISSUE THREAD

**Issue by guynwpg** [tomlandia/fuckr#61]
I can receive messages but other users don't seem to get the ones I send back :( 

It was working fine when I first starting using Fuckr but just recently this has been an issue.

I tried:
- logging out and back in
- trying different accounts
- trying the older version of Fuckr

Nothing fixes the problem :(

**Comment by GitHubGuy89** [comment]
Same problem. Receive messages but messages I send aren't being received.

**Comment by victorgrego** [comment]
have you guys tried to sniff your traffic? Try wireshark or fiddler.

**Comment by guynwpg** [comment]
Shouldn't have to go through the bother, it was working perfectly fine for days when I first started using Fuckr.

**Comment by GitHubGuy89** [comment]
Instrumedley seems to be having the same problem in an issue he posted also.
I'm using the Windows x64 version, if that helps narrow down the problem.

**Comment by mark-chat** [comment]
My chat is also not working.
64 bit windows.
I can receive messages just fine
I seem to "send" messages as usual but they are not transmitted and received by the recipient
Going through my message history, the last message I sent to which I had a response was 26th July.

Messages sent from 27th July onwards do not get sent (confirmed by messaging my iPhone profile)

**Comment by newtofuckr** [comment]
ive not had grindr work properly since early May. 

i had never used grindr prior to february 19th... and worked generally good for about 2 months and kept getting worse and worse with more my sent-msgs not getting to the intended members. ive given up on it. i havent even tried logging on since about early july. i check this site once every 5 days or so to see if there is a new update on the program.

i think tom has decided to neglect this project. i see on his profile that he is involved in other discussions on github, but hasnt made a contribution to fuckr for about 2 months. i wish he would authorize some other member(s) on here to take up the project. 

i use other websites in the meantime.

**Comment by guynwpg** [comment]
He's released the source code so it's easy for someone with the know how to pick it up

**Comment by ShawnAlexander** [comment]
Chat not working for me either. Most likely related to the latest Grindr update.

**Comment by kusaa** [comment]
Also having this issue... any updates on this from anybody?

**Comment by victorgrego** [comment]
Tom, I have been trying to reverse engineer grindr since I received the beta version. I found that they are migrating from eJabberd to MongooseIM and they architecture is solid now. The new app does not let me sniff it through common ways, as monitored proxys block the app to work. If you want to team up or need some help into coding feel free to poke me :)
Also, I am willing to update the docs, etc as soon as you get the app updated 
Thanks for updating Fuckr!

**Comment by shyrat** [comment]
Coincidentally I just spent the last night poking at the new 3.x version, and I noticed -- like you guys -- that it's going to be a lot more fun to reverse-engineer.  They appear to be rolling out a whole new REST API that hangs off of grindr.mobi.  It starts with a GET request to https://grindr.mobi/v3/bootstrap.  This only works if your user-agent is whitelisted, e.g., the official app uses "grindr3/3.0.0.4528;4528;Free;Android 4.2.2".

AFAICT, the official apps don't use the v2 API anymore.  So I suspect they're on the verge of shutting that one off.  I'd rather like to document the v3 API before that happens.  I'd be happy to collaborate with you guys on this.
